app/views/catalog/_search_targets_widget.html.erb

```diff
@@ -11,12 +11,15 @@
       <span class="caret"></span>
     </h1>
   </a>
-   <ul class="dropdown-menu" role="menu">
-     <li class="<%= 'active' unless article_search? %>">
-       <%= link_to_catalog_search %>
-     </li>
-     <li class="<%= 'active' if article_search? %>">
-       <%= link_to_article_search %>
-     </li>
+  <ul class="dropdown-menu" role="menu">
+    <li class="<%= 'active' unless article_search? %>">
+      <%= link_to_catalog_search %>
+    </li>
+    <li class="<%= 'active' if article_search? %>">
+      <%= link_to_article_search %>
+    </li>
+    <li>
+      <%= link_to_bento_search %>
```

Comment:
I believe the "All" link is supposed to be first: 

<img width="587" alt="desired-drop-down" src="https://user-images.githubusercontent.com/96776/30720463-745180e6-9edc-11e7-9e8b-67b54e375fbf.png">

Reply:
👍